assets/js/components/DatabaseDetails/DatabaseDetails.jsx

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
 import { useSelector } from 'react-redux';
-import ListView from '@components/ListView';
-import Table from '@components/Table';
-import {
-  databaseHostsTableConfiguration,
-  databaseInstancesTableConfiguration,
-} from './tableConfigs';
 import { getDatabaseDetail } from '@state/selectors';
+import GenericSystemDetails from '@components/SapSystemDetails/GenericSystemDetail';
```

Comment:
We should play a bit with `import` and `export` using `SapSystemDetails`'s `index.js` to allow this:

```js
import { GenericSystemDetails } from '@components/SapSystemDetails'
```

Reply:
I am able to achieve that by changing `./assets/js/components/SapSystemDetails/index.js` like this

```js
import { GenericSystemDetails as SystemDetails } from './GenericSystemDetails';
import SapSystemDetails from './SapSystemDetails';

export const GenericSystemDetails = SystemDetails;

export default SapSystemDetails;
```

I am a bit unsure about the aliasing `SystemDetails` which is needed to not make collide what we're importing with what  we're exporting (`GenericSystemDetails`).
Probably there's a better way of doing it, which I don't know by now :sweat_smile: 

But it works and allows us to do what you're looking for

```js
import { GenericSystemDetails } from '@components/SapSystemDetails'
```

Thoughts?

Reply:
I think I found a cleaner way to do it, without unnecessary aliasing

```js
import SapSystemDetails from './SapSystemDetails';

export { GenericSystemDetails } from './GenericSystemDetails';
export default SapSystemDetails;
```

feels better :smile:

Reply:
gone this way https://github.com/trento-project/web/pull/244/commits/e734303f8d56d7b0c18c2ef25d7194c7918b0ddb